### ISSUE TITLE
Ordering in Prismic

### ISSUE THREAD

**Issue by jennpb** [wellcometrust/wellcomecollection.org#2851]
## Type
- 🐛 Bug

We have a problem with list pages where content that has been migrated from a different system (Drupal, Wordpress), doesn't follow a chronological order. This is most apparent on /books and /articles.

@jamesgorrie has been in discussion with Prismic about this problem. They are aware of it and are working on fixing it properly, so we're waiting on them rather than trying a workaround on our platform.

Raising it here so we don't lose sight of it. It's essential we get a fix for this, otherwise we'll have to investigate other ways of hacking it.

**Comment by jamesgorrie** [comment]
Closing in favour of #2733 which I gave a better heading so it'll come up in search.
